mkosi.postinst.chroot

```diff
@@ -10,6 +10,10 @@ fi
 if [[ -d /etc/pam.d ]]; then
     find /etc/pam.d -mindepth 1 -exec mv {} /usr/lib/pam.d \;
     rmdir /etc/pam.d
+    if [[ -f /etc/pam.conf ]]; then
+        cp /etc/pam.conf /usr/lib/
+        rm -f /etc/pam.conf
```

Comment:
We have this amazing tool called `mv` you know

Reply:
But how do you know `mv` wasn't patched to break when used with /usr/ too?